lms/djangoapps/notification_prefs/views.py

```diff
@@ -15,6 +25,8 @@
 from openedx.core.djangoapps.user_api.models import UserPreference
 from openedx.core.djangoapps.user_api.preferences.api import delete_user_preference
 
+AES_BLOCK_SIZE_BYTES = int(AES.block_size / 8)
```

Comment:
Is `AES.block_size` guaranteed to be byte-aligned even though its units are bits?